src/UnitTests/test_quote_prov.cpp

```diff
@@ -33,50 +42,99 @@ static sgx_ql_set_logging_function_t sgx_ql_set_logging_function;
 
 static void Log(sgx_ql_log_level_t level, const char* message)
 {
-    char const * levelText = "ERROR";
-    switch (level) {
+    char const* levelText = "ERROR";
+    switch (level)
+    {
         case SGX_QL_LOG_WARNING:
             levelText = "WARNING";
             break;
         case SGX_QL_LOG_INFO:
             levelText = "INFO";
             break;
         default:
-	    levelText = "ERROR";
+            levelText = "ERROR";
     }
-    printf(
-        "[%s]: %s\n",
-        levelText,
-        message);
+    printf("[%s]: %s\n", levelText, message);
 }
 
+#if defined __LINUX__
 static void* LoadFunctions()
 {
-    // if this assert fails, be sure libdcap_quoteprov.so is in your LD_LIBRARY_PATH
+    // if this assert fails, be sure libdcap_quoteprov.so is in your
+    // LD_LIBRARY_PATH
+
     void* library = dlopen("libdcap_quoteprov.so", RTLD_NOW);
+    printf(dlerror());
     if (library == nullptr)
     {
         Log(SGX_QL_LOG_ERROR, dlerror());
         abort();
     }
 
-    sgx_ql_free_revocation_info = reinterpret_cast<sgx_ql_free_revocation_info_t>(dlsym(library, "sgx_ql_free_revocation_info"));
+    sgx_ql_free_revocation_info =
+        reinterpret_cast<sgx_ql_free_revocation_info_t>(
+            dlsym(library, "sgx_ql_free_revocation_info"));
     assert(sgx_ql_free_revocation_info);
 
-    sgx_ql_get_revocation_info = reinterpret_cast<sgx_ql_get_revocation_info_t>(dlsym(library, "sgx_ql_get_revocation_info"));
+    sgx_ql_get_revocation_info = reinterpret_cast<sgx_ql_get_revocation_info_t>(
+        dlsym(library, "sgx_ql_get_revocation_info"));
     assert(sgx_ql_get_revocation_info);
 
-    sgx_ql_free_quote_config = reinterpret_cast<sgx_ql_free_quote_config_t>(dlsym(library, "sgx_ql_free_quote_config"));
+    sgx_ql_free_quote_config = reinterpret_cast<sgx_ql_free_quote_config_t>(
+        dlsym(library, "sgx_ql_free_quote_config"));
     assert(sgx_ql_free_quote_config);
 
-    sgx_ql_get_quote_config = reinterpret_cast<sgx_ql_get_quote_config_t>(dlsym(library, "sgx_ql_get_quote_config"));
+    sgx_ql_get_quote_config = reinterpret_cast<sgx_ql_get_quote_config_t>(
+        dlsym(library, "sgx_ql_get_quote_config"));
     assert(sgx_ql_get_quote_config);
 
-    sgx_ql_set_logging_function = reinterpret_cast<sgx_ql_set_logging_function_t>(dlsym(library, "sgx_ql_set_logging_function"));
+    sgx_ql_set_logging_function =
+        reinterpret_cast<sgx_ql_set_logging_function_t>(
+            dlsym(library, "sgx_ql_set_logging_function"));
     assert(sgx_ql_set_logging_function);
 
     return library;
 }
+#else
+static HINSTANCE LoadFunctions()
+{
+    // https://stackoverflow.com/questions/27220/how-to-convert-stdstring-to-lpcwstr-in-c-unicode
+    std::wstring s = L"dcap_quoteprov.dll";
+    HINSTANCE hLibCapdll = LoadLibrary(s.c_str());
+    if (hLibCapdll == NULL)
+    {
+        DWORD error = GetLastError();
+        std::cout << error << std::endl;

```

Comment:
remove all references to std::cout and replace with Log(..).

Reply:
done